staff/aleix-caminal/cv/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+    <head>
+        <meta charset="utf-8">
+        <title>Curriculum Vitae</title>
+    </head>
+    <body>
+        <main>
+            <section id="head">
+                <div class="image">
+                    <img src="assets/img/profile.jpg" alt="profile">
+                </div>
+                <div class="info">
```

Comment:
Why did you decide to go for a `<div>` instead of using `<aside>` or `<section>`  

Check out the Content Sectioning: [RFM](https://developer.mozilla.org/en-US/docs/Web/HTML/Element)